geosolutions-it/MapStore2
# #7304 Aligned standard template to latest changes

## Description
Need to do a final test. 

Anyway I proceeded this way:
- I generated a project, applied fixes and ported them in template files
- Did a diff of the previous template files and the current ones to create the migration guidelines, that are long but well detailed

Tests to do:
 - Create a project and verify that project work, and that the application can work with extensions
 - Setup data dir and check if everything works as expected, as for the product.

I noticed that some of the fixes are missing also on master, so we should do a similar PR on master (also to port the Migration guidelines). But the things changes because of all updated to geostore, so we need a different PR. 
If on the next release we use the new project system, doing this additional work may become obsolete.
 
**Please check if the PR fulfills these requirements**


**What kind of change does this PR introduce?** (check one with "x", remove the others)



## Issue

**What is the current behavior?**

#7304

**What is the new behavior?**
Project created with `createProject now are working, and we can install extensions in them
Migration guidelines has been updated, so we can update existing project with the new features.


## Breaking change
**Does this PR introduce a breaking change?** (check one with "x", remove the other)



## Other useful information
